src/main/java/Remy/Remy.java

```diff
@@ -0,0 +1,60 @@
+package remy;
+
+import remy.command.Command;
+import remy.task.TaskList;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+// CS2103T Website Increment description-reused
+// Reused the example code from the website.
+public class Remy {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+
+    /**
+     * Sets up a Remy chatbot for use.
+     * @param filePath location to store tasklist after ending each session.
+     */
+    public Remy(Path filePath) {
+        ui = new Ui();
```

Comment:
I noticed `ui` is instantiated, but no instance methods of `ui` is invoked. Perhaps you might want to change the existing static methods to instance method. This will allow for easier unit testing of `Ui` as well, since static methods cannot be overridden to create stubs.